integrations/Vero/browser.js

```diff
@@ -34,21 +35,65 @@ class Vero {
     return !!window._veroq && !!window._veroq.ready;
   }
 
+  /**
+   * AddOrRemoveTags.
+   *
+   * http://developers.getvero.com/?javascript#tags
+   *
+   * @api public
+   * @param {Object} tags
+   */
+  addOrRemoveTags(tags, userId) {
+    const addTags = get(tags, "add");
+    const removeTags = get(tags, "remove");
+    window._veroq.push([
+      "tags",
+      {
+        id: userId,
+        add: addTags,
+        remove: removeTags,
+      },
+    ]);
+  }
+
+  /**
+   * Identify.
+   *
+   * https://developers.getvero.com/?javascript#users-identify
+   *
+   * @api public
+   * @param {Identify} identify
+   */
   identify(rudderElement) {
-    const message = { rudderElement };
-    const userId = message.userId || message.anonymousId;
+    const { message } = rudderElement;
     const { traits } = message.context || message;
-    // userId OR email address are required by Vero's API. When userId isn't present,
-    // email will be used as the userId.
+    const userId = message.userId || message.anonymousId;
+    /*
+      userId OR email address are required by Vero's API. When userId isn't present,
+      email will be used as the userId.
+     */
     const email =
       get(message, "context.traits.email") || get(message, "traits.email");
     if (!userId && !email) {
       logger.error("[Vero]: User parameter userId or email is required.");
       return;
     }
-    window._veroq.push([userId, { traits }]);
+    let payload = traits;
+    if (userId) payload = { id: userId, ...payload };
+    window._veroq.push(["user", payload]);
+    const tags = message.context.integerations?.Vero?.tags;
```

Comment:
Integrations object can contain `VERO`, `vero` like this as well so there should be support of canonical names. Check DCM impl

Reply:
Done